coordinator/userapi.go

```diff
@@ -54,6 +60,12 @@ func newUserAPIServer(mSGetter manifestSetGetter, caGetter certChainGetter, reg
 		),
 	)
 
+	manifestGeneration := promauto.With(reg).NewCounter(prometheus.CounterOpts{
```

Comment:
Using a counter might be incorrect here, considering coordinator restarts or distributed coordinators. Should this be a gauge instead?